students/runtests.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+"""Framework for tests."""
+import importlib
+import inspect
+import os
+import pathlib
+import sys
+import traceback
+
+
+def path_select(dir_name):
+    """Returns True dir exists on a computer."""
+    return os.path.isdir(dir_name)
+
+
+def file_select(dir_name, file1):
+    """Returns True if file meets requirements."""
+    path1 = '{0} + {1} + {2}'.format(dir_name, '/', file1)
+    if (pathlib.Path(path1).suffix == '.py'):

```

Comment:
You don't need `()` around expressions in python. See https://github.com/wemake-services/wemake-python-styleguide/issues/214